rhel7/checks/oval/service_all_disabled.xml

```diff
@@ -0,0 +1,31 @@
+<def-group>
+  <definition class="compliance" id="service_all_disabled" version="1">
+    <metadata>
+      <title>All services disabled</title>
```

Comment:
All Services Disabled In Container

Reply:
Which also means update xccdf/oval rule IDs